lk2nd/display/panel/generated/panels.h

```diff
@@ -5,6 +5,9 @@
 
 #include "lk_panel_hx8394d_720p_video.h"
 #include "lk_panel_ili9806e_boyi_byt45mp41t1m_4p5cpt.h"
+#include "lk_panel_ili9806e_fwvga_hsd_hlt_video.h"
+#include "lk_panel_ili9806e_fwvga_hsd_huashi_video.h"
+#include "lk_panel_ili9806e_fwvga_video.h"
```

Comment:
```suggestion
#include "lk_panel_ili9806e_fwvga_hsd_hlt_video.h"
#include "lk_panel_ili9806e_fwvga_hsd_huashi_video.h"
#include "lk_panel_ili9806e_fwvga_video.h"
```

You don't really have 3 panels on 1 phone at the same time. Remove the other 2 panels you don't have.